backend/blueprint/api/controllers/modifyUser.py

```diff
@@ -0,0 +1,45 @@
+from flask import Blueprint, jsonify, request
+from utils.response import Response
+from models.user import User
+from helpers.verifyPassword import verifyPassword
+from werkzeug.security import generate_password_hash
+
+router = Blueprint('modifyUser', __name__)
+
+
+@router.route('/password', methods=['PUT'])
+def changePassword():
+    data = request.get_json()
+    if data and data['username']:
+        user = User.getUser(data['username'])
+        if user:
+            if not verifyPassword(user, data['password']):
+                return jsonify(Response(401).generateMessage('Wrong username and password combination'))
+            user.password = generate_password_hash(data['newPassword'])
+            try:
+                user.save_to_db()
+            except Exception as e:
+                print(e)
```

Comment:
A better way would be to pass the exception as `str(e)` inside the `Response(401).generateMessage`?
What are your thoughts?

Reply:
Done :+1: